ethereum/blockwatch/block_watcher.go

```diff
@@ -78,20 +88,15 @@ type Watcher struct {
 }
 
 // New creates a new Watcher instance.
-func New(config Config) (*Watcher, error) {
-	checkpointID, err := config.Stack.Checkpoint()
-	if err != nil {
-		return nil, err
-	}
+func New(config Config) *Watcher {
 	bs := &Watcher{
```

Comment:
nit: This should probably just be called `watcher`. I'm not sure where the variable name `bs` came from but I'm guessing it made more sense at some point in the past.